scripts/deploy.sh

```diff
@@ -21,8 +21,31 @@ export BOOTSTRAP_OPTS=${BOOTSTRAP_OPTS:-""}
 
 OA_DIR='/opt/rpc-openstack/openstack-ansible'
 RPCD_DIR='/opt/rpc-openstack/rpcd'
-RPCD_VARS='/etc/openstack_deploy/user_extras_variables.yml'
-RPCD_SECRETS='/etc/openstack_deploy/user_extras_secrets.yml'
+RPCD_VARS='/etc/openstack_deploy/user_rpco_variables_defaults.yml'
+RPCD_SECRETS='/etc/openstack_deploy/user_rpco_secrets_defaults.yml'
+
+SECRETS_OVERRIDE_CONTENT='
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Using this file
+#
+# This file is laid down only once and is used for overriding variables based on the file name.
+# This file will not be overwritten by upgrades.
+
+noopvar: foobar'
```

Comment:
If everyone's happy with the idea of a dummy var. It might be worth adding a comment explaining its purpose, possibly linking to an upstream bug (assuming OSA accept that it shouldn't fail if an empty file is provided).